src/main/webapp/user/UserRepresentation.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * This class is responsible for displaying an icon for the creator of
+ * placeguides, used in each placeguide info box. The icon creates a link to
+ * the portfolio of the given user.
+ */
+class UserRepresentation {
+  constructor(user) {
+    this._userRepresentationDiv = UserRepresentation.createUserDiv(user);
+  }
+
+  get div() {
+    return this._userRepresentationDiv;
+  }
+
+  static createUserDiv(user) {
+    const userDiv = document.createElement("div");
+    userDiv.style.height = "5px";
+    userDiv.style.width = "50px";
+    const userDivClass = "userRepresentation-" + "{" + user.id + "}";
+    userDiv.setAttribute("class", userDivClass);
+    if (user.imgKey != undefined) {
+      if (user.name != undefined) {
+        userDiv.appendChild(
+            UserRepresentation.createUserImg(user.imgKey, user.name));
+      } else {
+        userDiv.appendChild(
+            UserRepresentation.createUserImg(user.imgKey, user.email.substring(
+                0, user.email.indexOf("@"))));
```

Comment:
Maybe extract this `user.email.substring(...)` into a helper method with a descriptive name, since we are duplicating it below as well.

Reply:
Done.